pkg/resource/resource_share/hooks.go

```diff
@@ -232,3 +232,120 @@ func (rm *resourceManager) getPermissionArns(ctx context.Context, r *resource) (
 
 	return nil
 }
+
+func (rm *resourceManager) syncResources(
```

Comment:
`syncResourceShareReources`? i know it's weird, but `syncResources` insinuate kubernetes resources..